### ISSUE TITLE
If we use the PicList, and double columns, it looks like the "span" and "col" meanings are reversed

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#170]
Here is a screen shot of a part of my GEN PicList file:

![image](https://user-images.githubusercontent.com/65363206/89944207-6b2c0300-dbed-11ea-8214-930a2c67b260.png)

Here are two screenshots of the resulting layouts: 
![image](https://user-images.githubusercontent.com/65363206/89944343-a1698280-dbed-11ea-9069-0aff62f5358d.png)
This FIRST one is Cain coming to kill Abel, which is clear marked as a "Span", so in a double-column setup, it should SPAN the whole page, correct?
 
 
![image](https://user-images.githubusercontent.com/65363206/89944285-8d258580-dbed-11ea-865f-4299d7c2cdb1.png)
This SECOND example page is the two instruments for GEN 4:21 (we trick PTXprint to allow two pictures on 1 verse by tweaking the referent anchor for one of them to the previous verse). The issue here is these are marked as "col" but they are formatted as SPAN pictures, going across both columns.

**Comment by mhosken** [comment]
@davidc86  Can you output the corresponding PrintDraft/tmpPicLists/01GEN....piclist file. That is what is really driving the TeX and we can tell from that whether the problem is in ptxprint or the TeX macros.

**Comment by markpenny** [comment]
Sorry to be a pain, but could you also regenerate your piclist files so that they are in the new USFM3-like format.

**Comment by markpenny** [comment]
After some further testing today I have discovered that the issue arises when there is a mismatch between size="col/span" and pgpos="t/b/tl/tr/bl/br" - My understanding is that the macros should at least drop the l/r if "span" is used, and it was working but doesn't seem to be any more.

**Comment by davidc86** [comment]
I've rebuilt the list (using v1, btw, congrats!). And I see an issue: 
If I choose **_not_** to include a Reference in the \fig field, PTXprint has troubles: 
1) it doesn't create an entry line for that picture in the PicList file, and 
2) it messes up the following \fig by putting part of its Alt text into its scr="...." section messing up that picture's file name.

So that are some issues I've figured out while try to figure out why Acts won't print now. (It has been printing just fine for over a month now.)  PTXprint gives no information, but I can tell it bails on the process right when it is about to put blue in the progress indicator. (Right before it says "Not responding") I'm pretty stumped on this one. It prints just fine with the older PicList format. Weird eh?
